docs/src/test/java/docs/http/javadsl/server/testkit/TestKitWithActorTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright (C) 2009-2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.http.javadsl.server.testkit;
+
+//#testkit-actor-integration
+import akka.actor.testkit.typed.javadsl.TestProbe;
+import akka.actor.typed.ActorSystem;
+import akka.actor.typed.javadsl.Adapter;
+import akka.http.javadsl.model.HttpRequest;
+import akka.http.javadsl.testkit.JUnitRouteTest;
+
+import akka.http.javadsl.testkit.TestRoute;
+import akka.http.javadsl.testkit.TestRouteResult;
+import org.junit.Test;
+
+public class TestKitWithActorTest extends JUnitRouteTest {
+
+    @Test
+    public void returnPongForGetPing() {
+        ActorSystem<Void> system = Adapter.toTyped(system());
```

Comment:
Move it to be a field?
Maybe a different name would help people to distinguish `system()` and `system`.

Reply:
Unfortunately it can't be set in the constructor, as `system()` is not populated yet at that point... we'd either have to make it a method or a field that initialized in the `@Before`, neither of which seem like an improvement.